docker-compose.yml

```diff
@@ -10,9 +10,9 @@ services:
   web:
     build:
       context: .
-      cache_from:
-        - batdevcontainerregistry.azurecr.io/manage-courses-backend:latest
-    image: batdevcontainerregistry.azurecr.io/manage-courses-backend:latest
+      #cache_from:
+      #  - govukdfe/manage-courses-backend-poc:latest
```

Comment:
just delete it, we can always put it back

Reply:
Removed